dotnet-authserver/tests/TeacherIdentity.AuthServer.EndToEndTests/LegacyTrnJourney.cs

```diff
@@ -0,0 +1,261 @@
+using Microsoft.Playwright;
+using Moq;
+using TeacherIdentity.AuthServer.Models;
+using TeacherIdentity.AuthServer.Oidc;
+using TeacherIdentity.AuthServer.Services.DqtApi;
+
+namespace TeacherIdentity.AuthServer.EndToEndTests;
+
+public class LegacyTrnJourney : IClassFixture<HostFixture>
```

Comment:
Carved these out into their own type so they're easily removed when the legacy journey is nuked.